src/app.js

```diff
@@ -1 +1,36 @@
-// write code here
+const fs = require('fs');
+const path = require('path');
+
+function moveFiles(source, destination) {
+  if (!source) {
+    // eslint-disable-next-line no-console
+    console.error('No source provided');
+
+    return;
+  }
+
+  if (!destination) {
+    // eslint-disable-next-line no-console
+    console.error('No destination provided');
+
+    return;
+  }
+
+  const isDirectory =
+    destination.endsWith('/') ||
+    (fs.existsSync(destination) && fs.lstatSync(destination).isDirectory());
+  let destinationPath = destination;
+
+  if (isDirectory) {
+    destinationPath = path.join(destination, path.basename(source));
+  }
+
+  fs.rename(source, destinationPath, (error) => {
+    if (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
+    }
+  });
+}
```

Comment:
Consider adding a check to verify if the source file exists before attempting to move it. This can prevent errors when the source file does not exist.